### PR TITLE
Rebasing Commits from ROS 1 Branch

### PR DESCRIPTION
**Pull request by DLu**
Cherry pick PRs #47 and #48 to ROS 2.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,11 @@
...
+
+ 1. [Building a Visual Robot Model with URDF from Scratch](https://wiki.ros.org/urdf/Tutorials/Building%20a%20Visual%20Robot%20Model%20with%20URDF%20from%20Scratch) - Learn how to build a visual model of a robot that you can view in Rviz
+
+ 1. [Building a Movable Robot Model with URDF](https://wiki.ros.org/urdf/Tutorials/Building%20a%20Movable%20Robot%20Model%20with%20URDF) - Learn how to define movable joints in URDF
+
+ 1. [Adding Physical and Collision Properties to a URDF Model](https://wiki.ros.org/urdf/Tutorials/Adding%20Physical%20and%20Collision%20Properties%20to%20a%20URDF%20Model) - Learn how to add collision and inertial properties to links, and how to add joint dynamics to joints.
+
+ 1. [Using Xacro to Clean Up a URDF File](https://wiki.ros.org/urdf/Tutorials/Using%20Xacro%20to%20Clean%20Up%20a%20URDF%20File) - Learn some tricks to reduce the amount of code in a URDF file using Xacro
+
+See also [urdf_sim_tutorial](https://github.com/ros/urdf_sim_tutorial) for the final tutorial: [Using a URDF in Gazebo](https://wiki.ros.org/urdf/Tutorials/Using%20a%20URDF%20in%20Gazebo) - Preliminary tutorial on how to spawn and control your robot in Gazebo.
```
Comment (hidmic) [inline README.md:11]: @DLu hmm, it doesn't look like these tutorials apply to ROS 2.

Comment (DLu) [reply README.md:11]: [Look what you made me do](https://github.com/ros2/ros2_documentation/pull/2001)

**Review by hidmic — APPROVED** [review]
LGTM!

@DLu would you mind updating this PR title to squash-merge or cleaning up commits to rebase-merge? I don't mind either way.

**Comment by DLu** [comment]
How's that?

**Comment by hidmic** [comment]
@DLu I went ahead and signed it off for you. I hope you don't mind.

**Pull request merged**